jest.config.e2e.js

```diff
@@ -0,0 +1,21 @@
+const _ = require('lodash')
+const config = require('./jest.config')
+
+const overrides = {
+  setupTestFrameworkScriptFile: '<rootDir>/__test__/e2e/util/setup.js',
+  testMatch: ['**/__test__/e2e/**/*.js'],
+  testPathIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/__test__/e2e/util/', '<rootDir>/__test__/e2e/pom/']
+}
+const merges = {
+  // Merge in changes to deeper objects
+  globals: {
+    // This sets the BASE_URL for the target of the e2e tests (what the tests are testing)
+    BASE_URL: 'localhost:3000'
```

Comment:
should this pull from an .env?

Reply:
I thought the same and I tried. However, the test (client) process isn't the same as the target (server) process, so none of the .env variables are available to the test. I suppose we could look into adding a foreman file but it's yet another piece of the config puzzle.